calcom/cal.com
# fix: applying default brandColor in embeds

## What does this PR do?



Fixes #10497
Now the brand color that the user sets in Settings>Appearance will be default brand color in embeds.



## Type of change




## How should this be tested?
1. Set a brand color in Settings>Appearance.
2. Go to Event Types, and click on embed of any event.
3. The brand color here should be the one user selects in settings.

Images-
![s1](https://github.com/calcom/cal.com/assets/44525862/1f7f94dd-d880-4185-8c3f-998937ad1119) ![s2](https://github.com/calcom/cal.com/assets/44525862/02da550f-81d6-407f-b01b-e0d75c3d1e38)

## Mandatory Tasks
